indico/modules/events/registration/util.py

```diff
@@ -217,29 +218,34 @@ def check_registration_email(regform, email, registration=None, management=False
             return dict(status='ok', user=None)
 
 
-def make_registration_form(regform, management=False, registration=None):
-    """Create a WTForm based on registration form fields."""
+def make_registration_schema(regform, management=False, registration=None):
+    """Dynamically create a Marshmallow schema based on the registration form fields."""
+    class RegistrationSchema(IndicoSchema):
+        class Meta:
+            unknown = EXCLUDE
```

Comment:
I'd keep the RAISE default so we quickly notice if we send something that's not expected/handled